x-pack/plugins/ml/public/application/jobs/new_job/pages/components/charts/event_rate_chart/overlay_range.tsx

```diff
@@ -39,6 +39,9 @@ export const OverlayRange: FC<Props> = ({ overlayKey, start, end, color, showMar
         style={{
           fill: color,
           strokeWidth: 0,
```

Comment:
Something in this change is causing the marker on this chart to disappear.

![image](https://user-images.githubusercontent.com/22172091/170242672-72adaed7-19f2-4a7e-9264-9e38bb0f6c41.png)

It should look lie this:

![image](https://user-images.githubusercontent.com/22172091/170242997-44dceafc-1de3-4ebe-849b-d761d80733d2.png)

Also hovering my mouse over the arrow icon causes the chart to fade out.